milli/src/search/query_tree.rs

```diff
@@ -155,10 +155,6 @@ impl fmt::Debug for Query {
 
 trait Context {
     fn word_docids(&self, word: &str) -> heed::Result<Option<RoaringBitmap>>;
-    fn stop_words(&self) -> anyhow::Result<Option<Set<&[u8]>>>;
-    fn is_stop_word(&self, word: &str) -> anyhow::Result<bool> {
-        Ok(self.stop_words()?.map_or(false, |s| s.contains(word)))
-    }
```

Comment:
I don't remember why these function have been removed from the context, and why we now use the index directly?

Reply:
it was removed by the revert of the first commit.
It was added so we could access the stop_words in the `resolve_primitive_part`, but we don't need it now.